### PR TITLE
[STACK 0.2] Add experimental to isort

### PR DESCRIPTION
**Pull request by SimonBiggs**
I changed the isort settings so that experimental imports are placed below normal pymedphys imports flagging their hierarchy with respect to the rest of the library.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -141,9 +141,10 @@ known_medphys = [
     "pydicom", "pylinac", "pynetdicom", "libjpeg"
 ]
 known_first_party = ["pymedphys"]
+known_experimental = ["pymedphys._experimental"]
 sections = [
     "FUTURE", "STDLIB", "THIRDPARTY", "GENERAL_SCIENTIFIC", "MACHINE_LEARNING",
-    "MEDPHYS", "FIRSTPARTY", "LOCALFOLDER"
+    "MEDPHYS", "FIRSTPARTY", "EXPERIMENTAL", "LOCALFOLDER"
```
Comment (SimonBiggs) [inline pyproject.toml:147]: This section here is the only piece that was actually updated. Everything else came from running `poetry run pre-commit run --all-files`.

**Review by sjswerdloff — APPROVED** [review]

**Comment by sjswerdloff** [comment]
it looks like some modules need to be committed/moved...

**Comment by SimonBiggs** [comment]
> it looks like some modules need to be committed/moved...

Yeah, oops. Fixed that in https://github.com/pymedphys/pymedphys/pull/1253.

**Comment by SimonBiggs** [comment]
> it looks like some modules need to be committed/moved...

A better practice would be to only mark for review once I see that the CI tests pass.

**Comment by sjswerdloff** [comment]
LGTM 👍

**Pull request merged**